source/whats-new.txt

```diff
@@ -12,6 +12,13 @@ What's New
    :depth: 1
    :class: singlecol
 
+.. tip:: Release Notes
+   
+   To learn more about changes and updates between versions, you can
+   read the `release notes
+   <https://github.com/mongodb/mongo-go-driver/releases>`__ published
```

Comment:
S: Rather than link to one specific driver, it might make sense to link to the [Tags page](https://github.com/mongodb/mongo-go-driver/tags). Clicking on any version from there takes you to the Release Notes.